README.md

```diff
@@ -25,6 +25,7 @@ Our iOS component library provides chat features like direct and group messaging
 
 ## Related documentation
 
+* [PubNub documentation for iOS Chat Components](https://www.pubnub.com/docs/chat/components/ios/overview-ios)
 * [Swift SDK Documentation](/docs/sdks/swift)
```

Comment:
Can this get updated to https://www.pubnub.com/docs/sdks/swift

Reply:
@CraigLn Sure, I updated all relative links in this README.md that were pasted from docs.